utam-compiler/src/main/resources/errors.config.json

```diff
@@ -388,5 +388,9 @@
   {
     "code": 3003,
     "message": "custom selector \"%s\" of the element \"%s\" is used for an element \"%s\" in the page object %s, but has a different type \"%s\""
+  },
+  {
+    "code": 4001,
+    "message": "property \"metadata\" is missing or misconfigured in the root"
```

Comment:
because it's a rule applicable for one page object, its code should start with 2 for consistency, 2006

Reply:
I wondered about that. I actually had it at that initially, but changed my mind. I'll revert.